mis_builder/models/mis_report_style.py

```diff
@@ -154,16 +155,49 @@ def merge(self, styles):
         return r
 
     @api.model
-    def render(self, lang, style_props, type, value):
+    def render(self, lang, style_props, type, value, currency=None):
         if type == 'num':
             return self.render_num(lang, value, style_props.divider,
                                    style_props.dp,
                                    style_props.prefix, style_props.suffix)
         elif type == 'pct':
             return self.render_pct(lang, value, style_props.dp)
+        elif type == 'mon':
+            return self.render_mon(lang, value, style_props.prefix,
+                                   style_props.suffix, currency)
         else:
             return self.render_str(lang, value)
 
+    @api.model
+    def render_mon(self, lang, value, prefix=None, suffix=None,
+                   currency=None):
+        # format number following user given or user currency
+        if value is None or value is AccountingNone:
+            return u''
+        if not currency:
+            currency = self.env.user.company_id.currency_id
+        elif isinstance(currency, int):
+            currency = self.env['res.currency'].browse(currency)
```

Comment:
this test for int is superfluous, IMO, lets force people to pass a proper currency object